h/models/annotation.py

```diff
@@ -136,6 +137,16 @@ class Annotation(Base):
         uselist=True,
     )
 
+    @property
+    def uuid(self):
+        """
+        Return the UUID representation of the annotation's ID.
+
+        Annotation IDs are stored in the DB as a UUID but represented in the app
+        and API in a different format. This property returns the UUID version.
+        """
+        return UUID(types.URLSafeUUID.url_safe_to_hex(self.id))
```

Comment:
I looked into this but didn't find any places where there was exactly this code. What code has tended to do is to log the URL-safe annotation IDs (ie. `annotation.id`), since that is what is conveniently available, and left it as an exercise for the reader to map those to UUIDs if they are trying to find the corresponding DB entries.